### ISSUE TITLE
Source not there and cannot find it: paragon-ntfs

### ISSUE THREAD

**Issue by mnencia** [caskroom/homebrew-cask#26585]
[`paragon-ntfs`](https://github.com/caskroom/homebrew-cask/blob/master/Casks/paragon-ntfs.rb)

It looks like that the paragon-ntfs installer has changed even if the version is still `14`. There is no `FSInstaller.app/Contents/Resources/Paragon NTFS for Mac OS X.pkg` any more. 

The current content of the package is:

```
$ ls /Volumes/ParagonFS.localized/FSInstaller.app/Contents/Resources/ -la
total 6880
drwxr-xr-x 32 mnencia staff    1088 Nov  1 08:48 .
drwxr-xr-x  8 mnencia staff     272 Nov  1 08:48 ..
-rw-r--r--  1 mnencia staff    1862 Nov  1 08:48 ActivationWindowController.nib
-rw-r--r--  1 mnencia staff  637432 Nov  1 08:48 AppIcon.icns
drwxr-xr-x 14 mnencia staff     476 Nov  1 08:48 Base.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 ar.lproj
-rwxr-xr-x  1 mnencia staff  142496 Nov  1 08:48 com.paragon-software.installer
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 cs.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 da.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 de.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 el.lproj
drwxr-xr-x 18 mnencia staff     612 Nov  1 08:48 en.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 es.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 fi.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 fr.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 hu.lproj
-rw-r--r--  1 mnencia staff   22104 Nov  1 08:48 ico_facebook_login.tiff
-rw-r--r--  1 mnencia staff   78454 Nov  1 08:48 ico_paragon_logo.tiff
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 it.lproj
drwxr-xr-x 18 mnencia staff     612 Nov  1 08:48 ja.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 ko.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 nb.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 nl.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 pl.lproj
-rw-r--r--  1 mnencia staff 6148535 Nov  1 08:48 product.zip
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 pt-BR.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 pt-PT.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 ru.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 sv.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 tr.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 zh-Hans.lproj
drwxr-xr-x 16 mnencia staff     544 Nov  1 08:48 zh-Hant.lproj

$ unzip -t /Volumes/ParagonFS.localized/FSInstaller.app/Contents/Resources/product.zip
Archive:  /Volumes/ParagonFS.localized/FSInstaller.app/Contents/Resources/product.zip
    testing: Info.plist               OK
    testing: product.pkg              OK
No errors detected in compressed data of /Volumes/ParagonFS.localized/FSInstaller.app/Contents/Resources/product.zip.
```

I guess that the product.pkg inside the zip file is the file to execute, but I don't know any way to instruct homebrew-cask to use it.

**Comment by RogerThiede** [comment]
FYI, here is the brew error due to the change of the developer's program:

```
» brew cask install paragon-ntfs
Warning: The default Caskroom location has moved to /usr/local/Caskroom.

Please migrate your Casks to the new location and delete /opt/homebrew-cask/Caskroom,
or if you would like to keep your Caskroom at /opt/homebrew-cask/Caskroom, add the
following to your HOMEBREW_CASK_OPTS:

  --caskroom=/opt/homebrew-cask/Caskroom

For more details on each of those options, see https://github.com/caskroom/homebrew-cask/issues/21913.
==> Downloading http://dl.paragon-software.com/demo/ntfsmac14_trial_e.dmg
######################################################################## 100.0%
==> No checksum defined for Cask paragon-ntfs, skipping verification
==> Running installer for paragon-ntfs; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
Error: pkg source file not found: '/opt/homebrew-cask/Caskroom/paragon-ntfs/14/FSInstaller.app/Contents/Resources/Paragon NTFS for Mac OS X.pkg'
```

This change occurred because the cask is ```sha256 :no_check``` since the developer updates the installer in-place with the same filename.

**Comment by vitorgalvao** [comment]
Fixed. Unfortunately, now it requires manual intervention.

**Comment by RogerThiede** [comment]
Closed by 20b2978fd5490611d150d4b7a85bdcda44d85f78
